### PR TITLE
feat(live): use serlo.dev for production environment

### PR DESCRIPTION
**Pull request by inyono**
Cloudflare credentials etc. should still work since serlo.dev is on the same Cloudflare account as serlo.org

### CODE REVIEW

File: modules/cloudflare/versions.tf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+terraform {
+  required_version = ">= 0.12"
```
Comment (inyono) [inline modules/cloudflare/versions.tf]: Is there a reason that some of the modules have a `versions.tf` and other's don't?

Comment (marinusma) [reply modules/cloudflare/versions.tf]: Nope. We should introduce it everywhere.

File: modules/cloudflare/main.tf [hunk 2]
```diff
@@ -0,0 +1,66 @@
+variable "domain" {
+  description = "The root domain, e.g. serlo.org"
+}
+
+# A-Record
+resource "cloudflare_record" "a-record" {
+  domain  = var.domain
+  name    = "@"
+  value   = module.gcloud.staticip_regional_address
```
Comment (marinusma) [inline modules/cloudflare/main.tf]: could be var.VARIABLENAME

File: modules/cloudflare/main.tf [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  domain  = var.domain
+  name    = "hi"
+  value   = var.domain
+  type    = "CNAME"
+  proxied = true
+}
+
+# en.
+resource "cloudflare_record" "cname_en" {
+  domain  = local.domai
```
Comment (marinusma) [inline modules/cloudflare/main.tf]: domain = var.domain

File: modules/cloudflare/versions.tf (hunk 1 above)
Comment (inyono) [reply modules/cloudflare/versions.tf]: And we can't do that globally somehow I suppose?

Comment (marinusma) [reply modules/cloudflare/versions.tf]: @rbock44 Do you have an opinion?

Comment (inyono) [reply modules/cloudflare/versions.tf]: Moved the versions.tf to our entries, i.e. live/{production,staging,dev}

**Pull request merged**